solara/server/assets/style.css

```diff
@@ -112,6 +112,7 @@ div.highlight {
 
 .solara-autorouter-content {
     height: 100%;
+    overflow: auto;
```

Comment:
This seems to cause a different issue:
<img width="1133" alt="image" src="https://github.com/widgetti/solara/assets/5592797/72648420-2bb7-4467-be3e-f5256e664880">
Another, more different scroll bar

Reply:
The cause seems to be the fact that the 8px margins on the `Card`, together with height/width 100% push it beyond the bounds of the element containing it...

Reply:
Also causes this to happen with `.markdown-output` cells
<img width="242" alt="image" src="https://github.com/widgetti/solara/assets/5592797/a3bf4d6c-71d2-464f-a758-18d75917fda5">